core/include/drivers/regulator.h

```diff
@@ -20,8 +20,15 @@
 #define REGULATOR_ALWAYS_ON	BIT(0)
 /* Enables pull down mode. DT property: regulator-pull-down */
 #define REGULATOR_PULL_DOWN	BIT(1)
+/*
+ * regulator-boot-on:
+ * It's expected that this regulator was left on by the bootloader.
```

Comment:
by the first stage bootloader?

Reply:
Effective number of bootstages depends on the platform.